docs/source/release_notes.rst

```diff
@@ -4,6 +4,206 @@
 Release Notes
 *************
 
+pyMOR 2020.1 (July ??, 2020)
+----------------------------
+We are proud to announce the release of pyMOR 2020.1! Highlights of this release
+are support for non-intrusive model order reduction using artificial neural networks,
+the subspace accelerated dominant pole algorithm (SAMDP) and the implicitly restarted
+Arnoldi method for eigenvalue computation. Parameter handling in pyMOR has been
+simplified, and a new series of hands-on tutorials helps getting started using pyMOR
+more easily.
+
+Over 600 single commits have entered this release. For a full list of changes
+see `here <https://github.com/pymor/pymor/compare/2019.2.x...2020.1.x>`__.
+
+pyMOR 2020.1 contains contributions by Linus Balicki, Tim Keil, Hendrik Kleikamp
+and Luca Mechelli. We are also happy to welcome Linus as a new main developer!
+See `here <https://github.com/pymor/pymor/blob/master/AUTHORS.md>`__ for more
+details.
+
+
+Release highlights
+^^^^^^^^^^^^^^^^^^
+
+Model order reduction using artificial neural networks
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+With this release, we introduce a simple approach for non-intrusive model order
+reduction to pyMOR that makes use of artificial neural networks
+`[#1001] <https://github.com/pymor/pymor/pull/1001>`_. The method was first
+described in [HU18]_ and only requires being able to compute solution snapshots of
+the full-order |Model|. Thus, it can be applied to arbitrary (nonlinear) |Models| even when no
+access to the model's |Operators| is possible.
+
+Our implementation internally wraps `PyTorch <https://pytorch.org>`_ for the training and evaluation of
+the neural networks. No knowledge of PyTorch or neural networks is required to apply the method.
+
+
+New system analysis and linear algebra algorithms
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+So far, computing poles of an |LTIModel| was only supported by its
+:meth:`~pymor.models.iosys.LTIModel.poles` method which uses a dense eigenvalue
+solver and converts the operators to dense matrices.
+
+The new version adds :meth:`~pymor.algorithms.samdp.samdp`
+(`[#834] <https://github.com/pymor/pymor/pull/834>`_) and
+:meth:`~pymor.algorithms.eigs.eigs`
+(`[#880] <https://github.com/pymor/pymor/pull/880>`_) which can compute a subset
+of the eigenvalues which are dominant in some sense.
```

Comment:
@pmli, this sounds like samdp and eigs do similar things. Is this intended?

Reply:
Not really. I'll work on it a bit.

Reply:
@sdrave What do you think about the new version?

Reply:
See below.